src/graph.js

```diff
@@ -1,8 +1,80 @@
 /* eslint-disable */
-class Graph {
+/**
+ * #### Graphs
+
+  * Should have methods named `addNode`, `contains`, `removeNode`, `addEdge`, `getEdge`, and
+  * `removeEdge`
+  * `addNode(newNode, toNode)` should add a new item to the graph.  If `toNode` is given then
+  * the new node should share an edge with an existing node `toNode`.
+  * `contains(value)` should return true if the graph contains the given value.
+  * `removeNode(value)` should remove the specified value from the graph.
+  * `addEdge(fromNode, toNode)` should add an edge between the two specified nodes.
+  * `getEdge(fromNode, toNode)` should return `true` if an edge exists between the two specified
+  * graph nodes.
+  * `removeEdge(fromNode, toNode)` should remove the edge between the two specified nodes.
+ */
+
+class Node {
```

Comment:
Nice, I like how you encapsulated this functionality in another class. Makes for a clean implementation.